### ISSUE TITLE
"prepare-commit-msg" hook does not work for "git commit" without "-m <msg>"

### ISSUE THREAD

**Issue by raido** [bk201-/jira-prepare-commit-msg#211]
It seems `prepare-commit-msg` hook does not work with `git commit` via editor. 

**Dependencies**

`git version 2.25.0`

```
"husky": "~4.2.5",
"jira-prepare-commit-msg": "~1.4.1"
```

**Failure**

`JIRA prepare commit msg > TypeError: Cannot read property 'replace' of undefined`

**Solution**

Simply switch from `prepare-commit-message` -> `commit-msg` hook.

```
"husky": {
    "hooks": {
      "commit-msg": "jira-prepare-commit-msg"
    }
  }
```

**Comment by bk201-** [comment]
HI @raido!

I can't repro this issue, could you please give me some steps on how to reproduce?

**Comment by bk201-** [comment]
Fixed! Please update to 1.4.2

**Comment by raido** [comment]
I tried and it works.

However for my setup I'll keep using `commit-msg` hook. Reasoning being that then ticket number is always added as post action to commit message.

With `prepare-commit-msg` hook, one can simply delete the ticket from line prefix when using editor compile their message and it would never be added back automatically.
